framework/src/main/java/org/checkerframework/framework/source/SourceChecker.java

```diff
@@ -2009,6 +2010,43 @@ public final Set<String> getSupportedAnnotationTypes() {
     return this.suppressWarningsStringsFromOption;
   }
 
+  /**
+   * Returns the options passed to this checker and its immediate parent checker.
+   *
+   * @return the options passed to this checker and its immediate parent checker
+   */
+  private Map<String, String> getAllOptions() {
```

Comment:
@mernst This logic only works for the immediate parent checker, I could change the logic to work up to all parents.